### DISCUSSION TITLE
User Suspension

### DISCUSSION THREAD

**Discussion by martinbndr** [pelican-dev/panel#12] [Ideas]
Being able to suspend users in pelican panel.
Users suspended should be completly logged out and won´t be able to login anymore. Specific error message will show that.
Please also add api support for that.

**Comment by justinkluever** [comment]
It could be easier to implement a button on the profile (and api endpoint) to just suspend all servers attached to the user in a fast way in case it is required.

**Reply by martinbndr** [reply]
That does not take the same effect, users could still login, update their account details and stuff. If a user needs to be suspended he should really do nothing on the panel.

**Comment by lancepioch** [comment]
Follow the issue: https://github.com/pelican-dev/panel/issues/70
